docs/docusaurus.config.ts

```diff
@@ -108,6 +108,22 @@ const config: Config = {
       theme: prismThemes.github,
       darkTheme: prismThemes.dracula,
     },
+    algolia: {
+      // The application ID provided by Algolia
+      appId: "XKVLO8P882",
+
+      // Public API key: it is safe to commit it
+      apiKey: "e78ac2f60eba60a4c4a353c0702cc7bb",
```

Comment:
https://www.algolia.com/doc/guides/security/api-keys/#risks-of-api-key-exposure Is this safe?

Reply:
they say it's safe to commit, langchain does the same:

https://github.com/langchain-ai/langchain/blob/master/docs/docusaurus.config.js#L300

but if we're not comfortable with it i can always take it from an env var

Reply:
Does this API key link to your email?

Reply:
i believe it's only linked to the application, not individual email

Reply:
if you check the algolia settings page, do you see the same key?